File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/Kendaraan.java
package id.ac.uph.pbo.pertemuan01;

public class Kendaraan {
    protected String merek;
    protected int    kecepatan;

    public Kendaraan(String merek) {
        this.merek     = merek;
        this.kecepatan = 0;
    }

    public void jalan() {
        kecepatan = 60;
        System.out.println("[Kendaraan:" + merek + "] Melaju " + kecepatan + " km/j");
    }

    public void berhenti() {
        kecepatan = 0;
        System.out.println("[Kendaraan:" + merek + "] Berhenti.");
    }
}

File: PYTHON/Pertemuan 01/Kendaraan.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
class Kendaraan:
    def __init__(self, merek: str):
        self.merek = merek
        self.kecepatan = 0

    def jalan(self):
        self.kecepatan = 60
        print(f"[Kendaraan:{self.merek}] Melaju {self.kecepatan} km/j")

    def berhenti(self):
        self.kecepatan = 0
        print(f"[Kendaraan:{self.merek}] Berhenti.")

